sigstore-conformance/src/main/java/dev/sigstore/conformance/Main.java

```diff
@@ -129,9 +136,20 @@ private static VerifyArguments parseVerifyArguments(Arguments args) {
 
   private static void executeVerify(VerifyArguments args) throws Exception {
     final var verifier = KeylessVerifier.builder().sigstorePublicDefaults().build();
-    final var artifactByteSource = asByteSource(args.artifact.toFile());
-    final byte[] artifactDigest = artifactByteSource.hash(Hashing.sha256()).asBytes();
+    final byte[] artifactDigest = sha256(args.artifact);
     verifier.verifyOnline(
         artifactDigest, Files.readAllBytes(args.certificate), Files.readAllBytes(args.signature));
   }
+
+  private static byte[] sha256(Path path) throws IOException, NoSuchAlgorithmException {
+    byte[] buffer = new byte[1024];
+    MessageDigest digest = MessageDigest.getInstance("SHA-256");
+    BufferedInputStream in = new BufferedInputStream(Files.newInputStream(path));
```

Comment:
Technically speaking, BufferedInputStream is not required since you use byte[1024] buffer anyway. It would be nice to have try-with-resources to close the stream.
However, those are tiny comments.

Reply:
No I was lazy. I'll fix this.